logstash-output-pagerduty.gemspec

```diff
@@ -1,7 +1,7 @@
 Gem::Specification.new do |s|
 
   s.name            = 'logstash-output-pagerduty'
-  s.version         = '1.0.0'
+  s.version         = '1.1.0'
```

Comment:
kind of controversial topic, because actually no test is an issue, but also adding a new feature to the project, jejejej! I have to admit I actually think we can avoid raising the version, however for some integration test purposes we really need to release a new version.

I will update to the one you requested.